source/mainnet/smart-contracts/tutorials/piggy-bank/deploying.rst

```diff
@@ -0,0 +1,212 @@
+.. _Rust: https://www.rust-lang.org/
+
+.. _piggy-bank-deploying:
+
+=======================================
+Deploying the piggy bank smart contract
+=======================================
+
+This is the fourth :ref:`part of a tutorial<piggy-bank>` on smart contract
+development.
+So far you have written and tested a piggy bank smart contract in the Rust_ programming language.
+
+Furthermore, you set up your testnet node on a server, created an account that is funded with some CCD, and imported your account key into the ``concordium-client`` configuration.
+
+This part focuses on how you can deploy your developed piggy bank smart contract to the Concordium testnet and interact with it.
+
+
+.. warning::
+
+   The reader is assumed to have basic knowledge of what a blockchain and smart
+   contract is, and some experience with Rust_.
+
+
+Preparation
+===========
+
+Before you start, make sure you completed the previous parts of the tutorial and have a running testnet node that has caught up to the head of the blockchain and an account with some CCD setup.
+
+Building a wasm module
+======================
+
+You have already built a wasm module from your piggy bank smart contract in a :ref:`previous part<cargo-concordium-build>` of this tutorial.
+
+Repeat this step since you might have done some changes to the smart contract code in :ref:`part 2<piggy-bank-testing>` of this tutorial. Go to the folder where you saved your piggy bank smart contract and use ``cargo concordium build`` to create a wasm module.
+
+.. code-block:: console
+
+   $cargo concordium build --out piggy_bank_part2.wasm.v1
+
+.. note::
+
+   This command outputs the module ``piggy_bank_part2.wasm.v1`` to the project's root folder.
+
+.. note::
+
+   Alternatively, you can find the module in the folder ``./target/concordium/wasm32-unknown-unknown/release/``.
+   Use the wasm module file suffixed with ``v1`` when deploying to the chain. The ``piggy_bank_part2.wasm`` file (without the ``v1`` suffix) is not the ``v0`` module, but rather the raw wasm module produced by cargo.
```

Comment:
I think if you supply `--out` then the `.v1` file will not be in `./target/concordium/wasm32-unknown-unknown/release`.

Reply:
I updated the sentence to:
`When you use the above command without the ``--out`` flag, you can find the module ... `